<!--%Tag(FULL)%-->
<launch>
	
	<param name="bextract_args" value="-mfcc" />
  <param name="feature_subs" value="1" />
  <param name="filter_peaks" value="false"/>
  <param name="input_mode" value="live" />
  <param name="gesture_topic" value="gesture_output" />
  <param name="music_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/music" />
  <param name="pca_dimension" value="4" />
  <param name="pca_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/pca" />
  <param name="sample_duration" value="2" />
  <param name="sample_log" value="100" />
	
	<node pkg="audiogesture" type="pca_extractor" name="PCAExtractor" output="screen"/>
	
</launch>
<!--%EndTag(FULL)%-->
